src/rust/ide/view/src/debug_scenes/visualization.rs

```diff
@@ -98,8 +103,9 @@ fn init(app:&Application) {
         &*class.signature.name == "Graph"
     }).expect("Couldn't find Graph class.");
     let visualization = vis_class.new_instance(&scene).expect("Couldn't create visualiser.");
+    visualization.activate.emit(());
```

Comment:
if visualization is using new frp syle, use `visualization.activate()` instead :)

Reply:
Not yet unfortunately. I would refactor it after merging your PR.